sdk/identity/azure-identity/azure/identity/_credentials/browser.py

```diff
@@ -64,17 +73,17 @@ def _request_token(self, *scopes, **kwargs):
 
         # start an HTTP server to receive the redirect
         server = None
-        redirect_uri = self._redirect_uri
-        if redirect_uri:
+        if self._parsed_url:
             try:
-                server = self._server_class(redirect_uri, timeout=self._timeout)
+                redirect_uri = "http://{}:{}".format(self._parsed_url.hostname, self._parsed_url.port)
```

Comment:
Really nitpicky, but should this line be moved into the `except` block since `redirect_uri` isn't used outside the error message?

Reply:
It's not in the diff, but this is used on line 99 as an argument to msal's `get_authorization_request_url()`.